test/example_data.dart

```diff
@@ -195,6 +195,20 @@ Reaction unicodeEmojiReaction = Reaction(
   userId: selfUser.userId,
 );
 
+Reaction realmEmoji = Reaction(
```

Comment:
```suggestion
Reaction realmEmojiReaction = Reaction(
```

This object isn't just an emoji, but an emoji reaction. In particular it specifies a user ID, which isn't part of specifying an emoji.